unittests/tools/test_pwn_sast_parser.py

```diff
@@ -31,3 +31,10 @@ def test_one_dup_finding(self):
         findings = parser.get_findings(testfile, Test())
         self.assertIsInstance(findings, list)
         self.assertEqual(1, len(findings))
+
+    def test_title_is_not_none(self):
+      testfile = open("unittests/scans/pwn_sast/one_finding.json")
+      parser = PWNSASTParser()
+      findings = parser.get_findings(testfile, Test())
+      self.assertIsInstance(findings, list)
+      assert findings.title is not None
```

Comment:
Test the value please
```suggestion
      self.assertEquals("...", findings.title)
```